src/pages/groupComparison/GroupComparisonUtils.tsx

```diff
@@ -894,3 +906,109 @@ export function getGroupsDownloadData(
+                >
+                    {getButtonNameWithDownPointer('Alteration Types')}
+                </button>
+            </DefaultTooltip>
+            <DefaultTooltip
+                trigger={['click']}
+                placement={'bottomRight'}
+                overlay={
+                    <SettingsMenu
+                        store={store}
+                        infoElement={alterationMenuHeader(
+                            store.hasMutationEnrichmentData,
+                            store.hasFusionEnrichmentData,
+                            store.hasCnaEnrichmentData,
+                            store.showDriverAnnotationMenuSection,
+                            store.showTierAnnotationMenuSection
+                        )}
+                        customDriverSourceName={
+                            getBrowserWindow().frontendConfig.serverConfig
+                                .oncoprint_custom_driver_annotation_binary_menu_label
+                        }
+                        showDriverAnnotationSection={
+                            store.showDriverAnnotationMenuSection
+                        }
+                        showTierAnnotationSection={
+                            store.showTierAnnotationMenuSection
+                        }
+                    />
+                }
+            >
+                <button
+                    data-test="AlterationEnrichmentAnnotationsSelectorButton"
+                    style={{
+                        marginLeft: '10px',
+                        marginBottom: '10px',
+                    }}
+                    className="btn btn-primary btn-sm"
+                >
+                    {getButtonNameWithDownPointer('Annotations')}
+                </button>
+            </DefaultTooltip>
+        </div>
+    ) : (
+        <AlterationEnrichmentTypeSelector
+            classNames={styles.inlineAlterationTypeSelectorMenu}
+            store={store}
+            handlers={handlers}
+            showMutations={store.hasMutationEnrichmentData}
+            showCnas={store.hasCnaEnrichmentData}
+            showFusions={store.hasMutationEnrichmentData}
+        />
+    );
+}
+
+export function alterationMenuHeader(
+    studyHasMutations: boolean,
+    studyHasFusions: boolean,
+    studyHasCnas: boolean,
+    showDriverSection: boolean,
+    showTiersSection: boolean
+): JSX.Element {
+    const sections = [];
+    studyHasMutations && sections.push('mutations');
+    studyHasFusions &&
+        (showDriverSection || showTiersSection) &&
+        sections.push('fusions');
+    studyHasCnas &&
+        (showDriverSection || showTiersSection) &&
+        sections.push('copy number alterations');
+    let text = sections.join(' and ');
+    if (sections.length === 3)
+        text = sections[0] + ', ' + sections[1] + ' and ' + sections[2];
+    return (
+        <span style={{ marginTop: 'auto', marginBottom: 'auto' }}>
```

Comment:
centering like this usually better handled with flexbox approach

Reply:
Ok, it turned out that this could simply be removed. So, updated.